kafka-ui-react-app/src/theme/theme.ts

```diff
@@ -127,17 +137,47 @@ const theme = {
     },
     chevronIconColor: Colors.neutral[50],
   },
-  thStyles: {
+  schemaStyles: {
     backgroundColor: {
-      normal: Colors.neutral[0],
+      tr: Colors.neutral[5],
+      div: Colors.neutral[0],
     },
-    color: {
-      normal: Colors.neutral[50],
-      hover: Colors.brand[50],
-      active: Colors.brand[50],
+  },
+  modalStyles: {
+    backgroundColor: Colors.neutral[0],
+    border: {
+      top: Colors.neutral[5],
+      bottom: Colors.neutral[5],
+    },
+  },
+  tableStyles: {
```

Comment:
I like how you nested all table properties there

Reply:
Not sure about all `*Styles` postfixes

It's just stylistic thing, so I think if we can always fix it after some discussion in another PR